.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Maybe add prettier as a dev dependency and add a prettify script, not everybody relies on ide extensions for formatting.

Reply:
True! I will consider it 😎